bin/i18n/codeorg_crowdin.yml

```diff
@@ -32,6 +32,7 @@ files: [
   # hourofcode content is handled by the hourofcode-specific sync
   "ignore" : [
     "/source/hourofcode/**",
+    "/source/dashboard/restricted.yml"
```

Comment:
Do we care about generalizing the file names similar to the hourofcode path, for future proofing?

Reply:
For now, it's only 1 file; and I think we will use just this 1 file for a while. Later, if we want to break it into more files we can change this config easily.

In addition, I put this file under `/source/dashboard/` path so it can be processed by the current in-up-down-out code without much changes.